src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -1,454 +1,27 @@
-      max-width: 560px;
-    }
-
-    .summary {
-      font-family: var(--primary-font-family);
-      font-style: normal;
-      font-weight: normal;
-      padding-bottom: var(--space-sm);
-      margin-bottom: var(--space-md);
-      font-size: var(--text-md);
-      max-width: 560px;
-      text-overflow: ellipsis;
-      color: var(--text-default);
-      overflow: hidden;
-      -webkit-line-clamp: 5;
-      -webkit-box-orient: vertical;
-      overflow: auto;
-      height: 150px;
-      line-height: 1.5;
-      padding-top: var(--space-md-fixed);
-      b {
-        font-weight: 700;
-      }
-    }
-
-    .info .smallTitle {
-      font-weight: bold;
-    }
-
-    .reportProblem {
-      align-self: flex-start;
-      margin-top: 1.2em;
-      color: var(--accent);
-
-      &:hover {
-        color: var(--accent-overlay);
-      }
-
-      & svg {
-        margin-inline-end: 0.4em;
-      }
-    }
-  }
-
-  .gameRequirements {
-    align-self: flex-start;
-    overflow: auto;
-  }
-
-  .gamePageStoreIcon {
-    fill: var(--text-default);
-    flex-grow: 1;
-    text-align: end;
-    padding-inline-end: var(--space-sm);
-
-    & > svg {
-      width: var(--text-5xl);
-      height: var(--text-5xl);
-      padding: var(--space-3xs);
-      border-radius: 10px;
-    }
-  }
-
-  .MuiLinearProgress-root {
-    width: 169px;
-    height: 5px;
-    background-color: var(--neutral-05);
-
-    .MuiLinearProgress-bar {
-      background-color: var(--success);
-    }
-  }
-
-  .selectFieldWrapper.Field ~ .buttonsWrapper {
-    margin-top: 24px;
-  }
-
-  .updateText {
-    cursor: pointer;
-
-    &:hover {
-      color: var(--text-secondary);
-    }
-  }
-
-  .pictureTimeContainer {
-    display: flex;
-    flex-direction: column;
-    max-height: 400px;
-  }
-
-  .infoWrapper.itemContainer {
-    place-content: center;
-    text-align: initial;
-    width: 20vw;
-    min-width: fit-content;
-    max-width: 290px;
-    margin-bottom: 12px;
-  }
-
-  .itemTitle {
-    font-weight: var(--bold);
-    text-align: center;
-  }
-
-  .checkBox {
-    display: flex;
-    margin-top: 6px;
-    align-items: center;
-  }
-
-  .dlcTitle {
-    font-weight: var(--bold);
-    margin-top: 16px;
-  }
-
-  .sdlTitle {
-    cursor: pointer;
-    font-weight: var(--bold);
-
-    &:hover {
-      color: var(--download-button);
-    }
-  }
-
-  .MuiIconButton-colorPrimary,
-  .MuiCheckbox-colorPrimary.Mui-checked {
-    color: var(--download-button);
-  }
-
-  .PrivateSwitchBase-root-1 {
-    padding: var(--space-3xs);
-  }
-
-  .buttonsWrapper .button {
-    min-width: max-content;
-  }
-
-  .buttonWithIcon {
-    display: flex;
-    place-items: center;
-    justify-content: center;
-
-    & > svg {
-      height: 20px;
-      margin-inline-end: 5px; /* for compatibility, before was hardcoded in JS*/
-    }
-  }
-
-  @keyframes animate-stripes {
-    100% {
-      background-position: -100px 0px;
-    }
-  }
-}
-
 .App:not(.oldDesign) .gameConfigContainer {
```

Comment:
if the removal of the old design is intentional, we can remove this `.App:not(.oldDesign)` selector